### ISSUE TITLE
Techlahoma

### ISSUE THREAD

**Issue by kacollins** [FreeCodeCampOKC/fccokc_web_legacy#17]
Add more information about Techlahoma and links to techlahoma.org and meetup.com/pro/Techlahoma.

**Comment by stu7** [comment]
I'd like to add this info to the about.html unless it belongs on the index.html?

**Comment by kacollins** [comment]
The about page would be a great place for it!

**Comment by stu7** [comment]
I'm working on this one next.

**Comment by mattstub** [comment]
@stu7 have you done any work on this? just wanting to clarify for people who may be looking for things to do for Hacktoberfest

**Comment by stu7** [comment]
@mstub yes, I did this earlier this month right before making a commit for it and a couple of other ones too. I thought my commits went through on the pull already. Might have to fix something holding it up. I've been out of town and state a few times this month, so I have t get back into the swing of things here.

**Comment by kacollins** [comment]
@stu7, let us know if you need any help getting this done!

**Comment by stu7** [comment]
@kacollins just working on getting the push to go through after making another change and commit.  Did a 'git pull upstream master' to sync.  Added the file, committed it and pushed it, but it was rejected. What am I missing?  Do I need to do a fetch here?

**Comment by mattstub** [comment]
@stu7 are you working off different branches? it looks like you've got a bunch of congested commits for different things and could be a problem?

**Comment by stu7** [comment]
@mstub I think I'm working from my fork, syncing with the master, making changes, adding the file(s), committing them and pushing? This is the first time I've run into this issue for pushes.  Yeah, I have a lot of small commits on this pull over the past month or longer. Its still open.  Git status returns ' On branch master'.  Your branch is ahead of origin/master by 15 commits. Use git push to publish your local  commits.

Here's what git is saying when a push is initiated and rejected.  
master -> master (fetch first)
failed to push some refs to http://github.com/stu7/fccokc_web.git
Updates were rejected because the remote contains work that you do not have locally. This is usually caused by another repository pushing to the same ref. You may first want to integrate the remote changes using (git pull) before pushing again.  **which I think I did** using 'git pull upstream master'
**it auto-merged several files with one file conflict** (I made the changes to it, committed and pushed it)
See the note about fast-forwards in git push --help for details.

**Comment by Jason9199** [comment]
@stu7 That error is because you're making changes on your master branch locally, and your remote master branch has changes that you don't have locally. You'll have to update your local branch with the changes already in your remote master branch with a `git pull`, and if there are conflicts resolve those first before it'll let you pull everything from the remote branch in. It's a little confusing doing things this way, and I'd like to suggest a better way of making changes to someone else's repo (FCCOKC in this case). I'll make an issue for updates to our CONTRIBUTING document on github to help with this.

First, you fork the repo of FCCOKC so you'll have your own copy of the code on your github. It sounds like you've already done this step. Secondly, create a branch on your repo for the changes you're wanting to make. Call it whatever you want, and then make your changes in that branch. Once you're done, create a pull request on the original repo (FCCOKC in this case), requesting to merge in the changes from your branch on your repo with the original source code hosted at FCCOKC.

From there on out, before you create a new branch to work on something new, do a `git pull upstream master` which will update your master to the latest. Then you're ready to make a new branch to work on something else, and you can follow the same steps outlined above.

One last thing, if you ever want to update your branch that you're working in with the changes in your master branch, you can run the command `git rebase master`. That will attempt to roll back your changes in your branch, update the branch with master, and then reapply your changes on top of it. As long as there's no merge conflicts that should go through without any issue.

If it were me, I would save a copy of the changes you've made either as a copy of the file you worked on or to notepad or something, then start over. By saving those changes somewhere on your machine you're not going to lose your work, you can paste them back in later. If you decide to do that, save your changes somewhere else, then delete the local copy of your branch and clone it again so you'll have all the latest from your repo. Then you can make a branch, make the changes in that branch, and open a pull request to add your changes to FCCOKC from that branch you made.

If I can help with this or anything further, please feel free to find me on Slack. If I'm offline I'll still get notified on my phone and be glad to help you through the process.

Thank you for working so hard on this! I promise, it gets easier. :)

**Comment by stu7** [comment]
Thanks Jason for the feedback. We finally got everything sorted after four hours on Slack, last night. It was somewhat of a rewind and refresh, but that's what it took. Matt was a big help in that process that couldn't have been accomplished in double that time if I was doing it solo. Matt had been in this spot more than once so his experience and guidance was critical to getting it fixed. We ended up having to delete my local repo and my fccokc fork on Github, then re-clone it and start from there.  One of the other big moves is to start creating and working from branches for each issue. Hopefully, I can minimize having to go through all this again.
